lib/generators/paper_trail/install/install_generator.rb

```diff
@@ -26,6 +26,12 @@ class InstallGenerator < MigrationGenerator
       default: false,
       desc: "Use uuid instead of bigint for item_id type (use only if tables use UUIDs)"
     )
+    class_option(
+      :json,
+      type: :boolean,
+      default: false,
+      desc: "Use json instead of text for object column (use only if database supports json)"
+    )
```

Comment:
What do you think about making JSON the default?

It's my understanding that all of Rails' built-in [connection adapters](https://github.com/rails/rails/tree/main/activerecord/lib/active_record/connection_adapters) support JSON columns. 

Our default for Postgres (only) should be JSONB.

We can have a --text option, or not. I'm fine either way.